docs/topics/downloader-middleware.rst

```diff
@@ -876,7 +876,7 @@ REDIRECT_MAX_TIMES
 Default: ``20``
 
 The maximum number of redirections that will be followed for a single request.
-After this maximum, the request's response is returned as is.
+After this maximum, the request returns :exc:`IgnoreRequest`.
```

Comment:
Here I do think we need to fix the wording, but I think we need a wording more akin to the one from the other section (before the change), e.g.

```suggestion
After this maximum, the request is dropped.
```